lib/rules/arrow-spacing.js

```diff
@@ -45,7 +45,7 @@ module.exports = function(context) {
 
     /**
      * Determines whether spaces before after arrow(`=>`) is satisfy rule.
-     * if before/after value is `true`, there should be one space.
+     * if before/after value is `true`, there should be spaces.
```

Comment:
"spaces" is plural, but in this case it should be simply more than zero.  I believe the most accurate is "one or more spaces".